src/pages/validation-rules-analysis/validation-rules-analysis-table/ValidationRulesAnalysisTable.js

```diff
@@ -75,6 +75,9 @@ class ValidationRulesAnalysisTable extends PureComponent {
                             validationRuleId={element.validationRuleId}
                             periodId={element.periodId}
                             organisationUnitId={element.organisationUnitId}
+                            attributeOptionComboId={
+                                element.attributeOptionComboId
+                            }
```

Comment:
Would this be better as a single line than 3 lines, for consistency with the preceding lines?

Reply:
Formatting is handled by [prettier](https://prettier.io/), so I basically have no say in that!